openedx/core/djangoapps/user_api/accounts/toggles.py

```diff
@@ -7,7 +7,7 @@
 from openedx.core.djangoapps.waffle_utils import WaffleFlag
 
 # .. toggle_name: REDIRECT_TO_ORDER_HISTORY_MICROFRONTEND
-# .. toggle_type: waffle_flag
+# .. toggle_implementation: WaffleFlag
```

Comment:
Can you fix the name to be the flag name?

Reply:
Sorry, I had done that originally but forgot to push that commit.